coredb-operator/src/controller.rs

```diff
@@ -40,6 +41,7 @@ pub static COREDB_FINALIZER: &str = "coredbs.coredb.io";
 #[cfg_attr(test, derive(Default))]
 #[kube(kind = "CoreDB", group = "coredb.io", version = "v1alpha1", namespaced)]
 #[kube(status = "CoreDBStatus", shortname = "cdb")]
+#[allow(non_snake_case)]
```

Comment:
Kubernetes yaml keys use camelCase.

Reply:
ah I see, it's rust that was having the issue with it